core/themes/basis/template.php

```diff
@@ -38,6 +38,37 @@ function basis_preprocess_page(&$variables) {
     $variables['classes'][] = 'view-name-' . $view->name;
   }
 
+  // Get the installed version.
+  $install_version = config_get('system.core', 'install_version');
+
+  // Process the version to retain major.minor.patch format.
+  if (!empty($install_version)) {
+    // Extract major, minor, and patch versions.
+    preg_match('/^(\d+)\.(\d+)(?:\.(\d+))?/', $install_version, $matches);
+
+    // Build the cleaned-up version.
+    if (!empty($matches)) {
+      $major = $matches[1];
+      $minor = $matches[2];
+      $patch = isset($matches[3]) && is_numeric($matches[3]) ? $matches[3] : '0';
+      $install_version = "$major.$minor.$patch";
+    }
+    else {
+      // Fallback in case of unexpected format.
+      $install_version = '0.0.0';
+    }
+  }
+
+  // Every time we add supplemental CSS we add the version number here.
+  $supplemental_css_versions = array('1.30.0');
```

Comment:
This list should probably be more prominent and global so it can be accessed in the theme settings and possibly elsewhere. I'm not quite sure where is best since this list is specific to Basis. One thought is to put it in `basis.info` like `supplemental_css[] = 1.30.2` since that does get loaded globally for the active theme (even for child themes).

Reply:
I agree it needs to be in a reusable location. Maybe for now we just put it in a function, `basis_supplemental_css_versions()`, and consider .info file if there's some reason why it needs to be more accessible. `basis_supplemental_css_versions()` could hard-code the list for now, and in the future potentially pull from the .info file.

I'm a little hesitant about using the .info file in this way, since this is basis-specific functionality. People may get the impression all themes would have that capability.